airbyte-integrations/connectors/destination-iceberg/build.gradle

```diff
@@ -31,6 +31,7 @@ dependencies {
         exclude(group: 'org.apache.hadoop', module: 'hadoop-common')
     }
     implementation 'org.apache.iceberg:iceberg-spark-runtime-3.3_2.13:1.3.0'
+    implementation 'org.projectnessie.nessie-integrations:nessie-spark-extensions-3.3_2.13:0.73.0'
```

Comment:
Iceberg 1.3.0 [includes](https://github.com/apache/iceberg/blob/apache-iceberg-1.3.0/versions.props#L25) Nessie Client 0.59.0. Spark Extensions may conflict with that in the client library. Even if they work fine now, compatibility with code compiled to _future_ client versions is not guaranteed. It may be best to align the Nessie Spark Extensions version to what is included in Iceberg.